code/datums/tutorial/xenomorph/abomination.dm

```diff
@@ -0,0 +1,214 @@
+/datum/tutorial/xenomorph/abomination
+	name = "Xenomorph - Abomination"
+	desc = "A tutorial to teach you how to play the Abomination xenomorph caste. Completing this is required to be able to play an Abomination."
+	icon_state = "predalien"
+	tutorial_id = "xeno_abom_1"
+	tutorial_template = /datum/map_template/tutorial/s7x7
+	starting_xenomorph_type = /mob/living/carbon/xenomorph/predalien/tutorial
+
+// START OF SCRITPING
+
+/datum/tutorial/xenomorph/abomination/start_tutorial(mob/starting_mob)
+	. = ..()
+	if(!.)
+		return
+
+	init_mob()
+	xeno.melee_damage_lower = 40
+	xeno.melee_damage_upper = 40
```

Comment:
Of note, this will be lost ever `recalculate_damage` occurs because it is the caste's values that matter. If `recalculate_damage` can occur, it would be better to have this done via a `damage_modifier` and or the tutorial caste to just have different values.

Reply:
Code's not really needed any more (mattered more for an earlier iteration), removed